File: Asteroid_API/Asteroid_API/Models/Quotes.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Asteroid_API.Models
{
    public class Quotes
    {
        [Key]
        public int Percentage { get; set; }
        public string Quote { get; set; }
        public string Recommendation { get; set; }
    }
}

File: Asteroid_Angular/Asteroid/projects/asteroid-app/src/app/Models/Quotes.ts
export class Quotes {
    constructor(
        public percentage: Number,
        public quote: String, 
        public recommendation: String,        
    ){}

    getPercentage(){
        return this.percentage
    }

    getQuote(){
        return this.quote
    }

    getRec(){
        return this.recommendation
    }
}
 